Python/crossinterp_data_lookup.h

```diff
@@ -435,6 +435,23 @@ _none_shared(PyThreadState *tstate, PyObject *obj,
     return 0;
 }
 
+// NotImplemented
+
+static PyObject *
+_new_notimplemented_object(_PyCrossInterpreterData *data)
+{
+    return Py_NewRef(Py_NotImplemented);
```

Comment:
I'd rather skip reference counting for immortals:
```suggestion
    assert(_Py_IsImmortal(Py_NotImplemented));
    return Py_NotImplemented;
```